src/test/java/org/isf/accounting/Tests.java

```diff
@@ -469,18 +469,18 @@ public void ioGetDistictsBillItems() throws Exception {
 	}
 
 	@Test
-	public void ioBillItemChecks() throws Exception {
+	void ioBillItemChecks() throws Exception {
 		int id = setupTestBillItems(false);
 		BillItems foundBillItem = accountingBillItemsIoOperationRepository.findById(id).orElse(null);
 		assertThat(foundBillItem).isNotNull();
 		List<BillItems> billItems = accountingIoOperation.getDistictsBillItems();
 		assertThat(billItems).hasSize(1);
 
 		BillItems billItem = billItems.get(0);
-		assertThat(foundBillItem).isEqualTo(foundBillItem);
 		assertThat(foundBillItem)
-				.isNotEqualTo(TimeTools.getNow())
-				.isEqualTo(billItem);
+			.isEqualTo(foundBillItem)
```

Comment:
is this equal redundant?

Reply:
It may look strange but it is there to force executing every statement in the `equals()` method.

Reply:
Changed.